### DISCUSSION TITLE
Questions on precoding/combining vectors.

### DISCUSSION THREAD

**Discussion by fanyimeng0** [NVlabs/sionna#445] [General]
Hello, I would like to know how precoding and combining  works, especially in the case of VH polarisation or cross polarisation, for example, I have a 4*4 antenna array with 32 antennas in VH polarisation settings, but the precoding is supposed to be a 16 dimensional tenserflow , how can I make the these two parts match？

**Comment by faycalaa** [comment]
Hi,

For [coverage_maps](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.coverage_map), the size of the precoding (combining) vector must match the number of transmit (receive) linearly polarized antennas. Dual-polarized antennas are counted as two linearly polarized antennas. Therefore, for a 4 $\times$ 4 transmit antenna array with VH polarization, the size of the precoding vector should be 32.

**Comment by fanyimeng0** [comment]
Thanks for your response.
